doc/aqd.rst

```diff
@@ -1,13 +1,17 @@
 Processing Aquadopp (currents) data
 ***********************************
 
-Data will generally be processed using a series of run scripts. The first script for each instrument type
-depends on two :doc:`configuration files </config>`.
+After getting the code, make sure you change sys.path or sys.path.append lines point to the equivalent place on the local system.
```

Comment:
Please wrap sys.path and sys.path.append in double backticks, so they look like this:

    ``sys.path``